schema-admin.graphql

```diff
@@ -86,6 +86,7 @@ input CreateCollectionStoryInput {
   authors: [CollectionStoryAuthorInput!]!
   publisher: String!
   sortOrder: Int
+  fromPartner: Boolean!
```

Comment:
similar question for the API - what do you think about making this optional?

Reply:
Done!